src/components/Book.js

```diff
@@ -0,0 +1,68 @@
+// Individual Book component
+// IMPORTS
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+import { RemoveBookButton } from './Button';
+import '../styles/book.css';
+import progress from '../Assets/progress.png';
+
+const Book = ({
+  id, title, author, category,
+}) => {
+  const dispatch = useDispatch();
+  const handleRemoveBook = () => {
+    dispatch(removeBook(id));
+  };
+  return (
+    <div className="book">
+      <div className="book-cover">
+        <div className="book-info">
+          <h6 className="category">
+            {category}
+            Action
+          </h6>
+          <h3 className="title">
+            {title}
+          </h3>
+          <p className="author">
+            {author}
+          </p>
+        </div>
+        <div className="book-actions">
+          <p className="comments">
+            Comments
+          </p>
+          <span className="remove">
+            <RemoveBookButton className="remove-btn" type="button" onClick={handleRemoveBook}>Remove</RemoveBookButton>
+          </span>
+          <p className="edit">
+            Edit
+          </p>
+        </div>
+      </div>
+      <div className="progress">
+        <img src={progress} alt="progress" />
+        <div className="progress-info">
+          <p className="parcent">64%</p>
+          <p className="completed">Completed</p>
+        </div>
+      </div>
+      <div className="chapter">
+        <p className="current">CURRENT CHAPTER</p>
+        <p className="chapter-number">Chapter 17</p>
+        <button type="button" className="update-progress"><p>UPDATE PROGRESS</p></button>
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are using the border colour which should be removed as per the `Zeplin` design etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the attached image and table below.

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/HossainAraf/book-store-spa/assets/66476329/c7b48940-c48a-4ce3-aed0-807983546486)

</td>
<td>

![image](https://github.com/HossainAraf/book-store-spa/assets/66476329/e1211f49-7d36-42b1-979f-fa1abc97813f)

</td>
</tr>
</table>